tests/test_au/tests.py

```diff
@@ -195,6 +196,56 @@ def test_raises_error_for_invalid_abn(self):
         self.assertRaises(ValidationError, lambda: validator(invalid_abn))
 
 
+class AULocalFlavorAUCompanyNumberFieldValidatorTests(TestCase):
+
+    def test_no_error_for_a_valid_acn(self):
+        """Test a valid ACN does not cause an error."""
+        valid_acn = '604327504'
+
+        validator = AUCompanyNumberFieldValidator()
+
+        validator(valid_acn)
```

Comment:
It would be nice to be consistent on the spacing. For instance, some tests have a space between each statement while one has no spaces. The space don't seem to be needed because it's pretty simple code. Feel free to fix the previous tests as well where this was copied from.